tests/functional/appsync/test_appsync_resolver_utils.py

```diff
@@ -0,0 +1,219 @@
+import asyncio
+import datetime
+import json
+import os
+import sys
+
+import pytest
+
+from aws_lambda_powertools.utilities.data_classes import AppSyncResolverEvent
+from aws_lambda_powertools.utilities.data_classes.appsync_resolver_utils import (
+    AppSyncResolver,
+    aws_date,
+    aws_datetime,
+    aws_time,
+    aws_timestamp,
+    make_id,
+)
+from aws_lambda_powertools.utilities.typing import LambdaContext
+
+
+def load_event(file_name: str) -> dict:
+    full_file_name = os.path.dirname(os.path.realpath(__file__)) + "/../../events/" + file_name
+    with open(full_file_name) as fp:
+        return json.load(fp)
+
+
+def test_direct_resolver():
+    # Check whether we can handle an example appsync direct resolver
+    mock_event = load_event("appSyncDirectResolver.json")
+
+    app = AppSyncResolver()
+
+    @app.resolver(field_name="createSomething", include_context=True)
```

Comment:
I wonder if this would be considered a Dataclass. I see two options:

1. Discuss some design options in a RFC for lightweight API handling
2. Expand Event Source Data Classes description to include utilities in it - Current description:`The event source data classes utility provides classes describing the schema of common Lambda events triggers.`

The first is my preference as we might find other use cases, UX, as we put this into written form.

Reply:
@heitorlessa  kind of started that here:
https://github.com/awslabs/aws-lambda-powertools-python/issues/324

But yes, in general having lightweight decorators help. Maybe like the SQS batch processing?

Reply:
It's missing the RFC body like summary, drawbacks, alternatives, etc ;) one step a time